schema.cue

```diff
@@ -0,0 +1,62 @@
+import (
+	"time"
+)
+
+// Date is a date in the format YYYY-MM-DD
+#Timestamp: time.Format(time.RFC3339)
+
+#TypeURI: string
+
+#BaselineControlID: =~"^OSPS-[A-Z]{2}-[0-9]{2}.[0-9]{2}$"
+
+#ResourceDescriptor: {
+	description: string
+	mediaType:   string
+	uri:         #TypeURI
+}
+
+#Ella: {
+	uri:     #TypeURI
+	uriData: _ // see https://cuelang.org/docs/tour/types/top/
+}
+
+#EllaEvidence: {
+	name?: string
+	ella:  #Ella
+}
+
+#HumanEvidence: {
+	name?:            string
+	humanDescription: string
+}
+
+#AttestationRefEvidence: {
+	name?: string
+	ref:   #ResourceDescriptor
+}
+
+#Evidence: {
+	ellaEvidence?:           #EllaEvidence
+	humanEvidence?:          #HumanEvidence
+	attestationRefEvidence?: #AttestationRefEvidence
+}
+
+#Control: {
+	id:         #BaselineControlID
+	conclusion: "compliant" | "non-compliant"
+	evidence: [#Evidence, ...#Evidence]
+}
+
+#Evaluator: {
+	id:      #TypeURI
```

Comment:
I think we want both a TypeURI for the _type_ of thing (human, Privateer, Minder, Darn, Scorecard, etc) evaluating the project, and a separate ResourceURI indicating the specific _instance_ of that type (email address, hosted URI, etc).

The _instance_ might also then sign over the evaluation with the same identity as the ResourceURI.